components/brave_wallet/common/brave_wallet_types.h

```diff
@@ -11,10 +11,23 @@
 #include <vector>
 
 #include "base/values.h"
+#include "brave/components/brave_wallet/common/brave_wallet.mojom.h"
 #include "third_party/abseil-cpp/absl/types/optional.h"
 
 namespace brave_wallet {
 
+namespace mojom {
+// Remove these aliases eventually.
+constexpr KeyringId kFilecoinKeyringId = KeyringId::kFilecoinKeyringId;
+constexpr KeyringId kFilecoinTestnetKeyringId =
+    KeyringId::kFilecoinTestnetKeyringId;
+constexpr KeyringId kSolanaKeyringId = KeyringId::kSolanaKeyringId;
+constexpr KeyringId kDefaultKeyringId = KeyringId::kDefaultKeyringId;
+constexpr KeyringId kBitcoinKeyring84Id = KeyringId::kBitcoinKeyring84Id;
+constexpr KeyringId kBitcoinKeyring84TestId =
+    KeyringId::kBitcoinKeyring84TestId;
+}  // namespace mojom
+
```

Comment:
This allows `mojom::kDefaultKeyringId` to have same meaning as before. Otherwise PR would have way more changes to review.
Will cleanup eventually.

Reply:
Please add a `FIXME(supermassive)` in that case.